packages/dom/src/serialize-cssom.js

```diff
@@ -62,6 +62,33 @@ export function serializeCSSOM({ dom, clone, resources, cache }) {
       clone.prepend(styleLink);
     }
   }
+
+  // Capture blob css assets
+  for (let styleSheet of dom.styleSheets) {
```

Comment:
We can do this in above loop for in memory CSSOM, no need to loop multiple times